src/components/shared/Entity/hooks/useEntityCreateNavigate.ts

```diff
@@ -7,19 +7,20 @@ import { ENTITY_SETTINGS } from 'settings/entity';
 import { EntityWithCreateWorkflow } from 'types';
 import { getPathWithParams, hasLength } from 'utils/misc-utils';
 
+export interface HookEntityCreateNavigateProps {
+    advanceToForm?: boolean;
+    dataPlaneId?: string | null;
+    id?: string | null | undefined;
+}
```

Comment:
`replace` was not being used so I removed it. It may make it harder to track back the original settings if we want to put it back in. However, I'd rather not keep it hanging around.